README.md

```diff
@@ -32,217 +33,27 @@ let package = Package(
-
-```
-git clone https://github.com/apple/swift.git
-./swift/utils/update-checkout --clone
-./swift/utils/build-parser-lib --release --no-assertions --build-dir /tmp/parser-lib-build
-```
-
-### Embedding in an iOS Application
-
-You need to build `lib_InternalSwiftSyntaxParser.dylib` yourself, you cannot copy it from the toolchain. Follow the instructions above and change the invocation of `build-parser-lib` accordingly:
-
-```
-./swift/utils/build-parser-lib --release --no-assertions --build-dir /tmp/parser-lib-build-iossim --host iphonesimulator --architectures x86_64
-./swift/utils/build-parser-lib --release --no-assertions --build-dir /tmp/parser-lib-build-ios --host iphoneos --architectures arm64
-```
-
-### Some Example Users
-
-[**Swift AST Explorer**](https://swift-ast-explorer.kishikawakatsumi.com/): a Swift AST visualizer.
-
-[**swift-format**](https://github.com/apple/swift-format): formatting technology for Swift source code.
-
-[**Swift Stress Tester**](https://github.com/apple/swift-stress-tester): a test driver for sourcekitd and Swift evolution.
-
-[**SwiftSemantics**](https://github.com/SwiftDocOrg/SwiftSemantics): parses Swift code into its constituent declarations.
-
-[**Sitrep**](https://github.com/twostraws/Sitrep): A source code analyzer for Swift projects
-
-[**SwiftRewriter**](https://github.com/inamiy/SwiftRewriter): a Swift code formatter.
-
-[**SwiftPack**](https://github.com/omochi/SwiftPack): a tool for automatically embedding Swift library source.
-
-[**Periphery**](https://github.com/peripheryapp/periphery): a tool to detect unused code.
-
-[**BartyCrouch**](https://github.com/Flinesoft/BartyCrouch): a tool to incrementally update strings files to help App localization.
-
-[**Muter**](https://github.com/muter-mutation-testing/muter): Automated mutation testing for Swift
-
-[**Swift Variable Injector**](https://github.com/LucianoPAlmeida/variable-injector): a tool to replace string literals with environment variables values.
-
-[**Pecker**](https://github.com/woshiccm/Pecker): a tool to detect unused code based on [SwiftSyntax](https://github.com/apple/swift-syntax.git) and [IndexStoreDB](https://github.com/apple/indexstore-db.git).
-
-[**Piranha**](https://github.com/uber/piranha): a tool for refactoring code related to feature flags.
-
-[**STAR**](https://github.com/thumbtack/star): a tool to find how often specified Swift type(s) are used in a project.
-
-### Reporting Issues
-
-If you should hit any issues while using SwiftSyntax, we appreciate bug reports on [bugs.swift.org](https://bugs.swift.org) in the [SwiftSyntax component](https://bugs.swift.org/issues/?jql=component%20%3D%20SwiftSyntax).
+Documentation can be found [here](https://github.com/apple/swift-syntax/blob/main/Documentation/README.md).
```

Comment:
Maybe also reference the examples:
```suggestion
Documentation can be found [here](https://github.com/apple/swift-syntax/blob/main/Documentation) and some examples of using SwiftSyntax can be found [here](https://github.com/apple/swift-syntax/blob/main/Examples).
```
I would also reference the documentation directory instead of the REAMDE. It still renders the README quite prominently but you also have access to the other files in the directory that way.